web/src/main/resources/org/springframework/security/web/messages_es.properties

```diff
@@ -0,0 +1 @@
+ExceptionTranslationFilter.insufficientAuthentication=Para acceder a este recurso se requiere autenticaci\u00f3n completa
```

Comment:
Please also add this translation to `core/src/main/resources/org/springframework/security/messages_es_ES.properties`.